src/main/java/nl/hsac/fitnesse/fixture/util/selenium/driverfactory/DriverManager.java

```diff
@@ -24,6 +24,10 @@ public DriverManager() {
         Runtime.getRuntime().addShutdownHook(new Thread(() -> closeDriver()));
     }
 
+    protected DriverManager(DriverFactory driverFactory) {
```

Comment:
Why is the extra constructor needed? I believe Mockito will also inject mocks directly into (private) fields, so it does not need it

Reply:
Apparently :-).